normalize_repos/labels.py

```diff
@@ -35,6 +35,10 @@
         'color': 'e4e669',
         'description': 'This doesn\'t seem right'
     },
+    'meta': {
+        'color': 'ff8c00',
```

Comment:
I think this color might be similar to the "CC staff only" color. We want labels to have distinctive colors.

Use this color: `#04338c`.

Reply:
@kgodey Thanks for the review.